mlflow/paddle/__init__.py

```diff
@@ -47,6 +47,7 @@ def get_default_conda_env():
 def save_model(
     pd_model,
     path,
+    training=False,
```

Comment:
Can we add this argument to `log_model` as well?

Reply:
ohh yes. Sorry for carelessness

Reply:
I will also add some new unit tests on ```log_model``` based on high level api.

Reply:
Done

Reply:
finished~